hal/halx86/generic/sysinfo.c

```diff
@@ -12,8 +12,50 @@
 #define NDEBUG
 #include <debug.h>
 
+NTSTATUS
+NTAPI
+HaliHandlePCIConfigSpaceAccess(BOOLEAN IsRead,
+                               ULONG Port,
+                               ULONG Length,
+                               PVOID Buffer);
+
+AMLI_ILLEGAL_IO_PORT_ADDRESSES AMLIIllegalIOPortAddresses[19] =
+{
+    { 0x0000, 0x10, 1, NULL },
+    { 0x0020, 0x02, 0, NULL },
+    { 0x0040, 0x04, 1, NULL },
+    { 0x0048, 0x04, 1, NULL },
+    { 0x0070, 0x02, 1, NULL },
+    { 0x0074, 0x03, 1, NULL },
+    { 0x0081, 0x03, 1, NULL },
+    { 0x0087, 0x01, 1, NULL },
+    { 0x0089, 0x01, 1, NULL },
+    { 0x008A, 0x02, 1, NULL },
+    { 0x008F, 0x01, 1, NULL },
+    { 0x0090, 0x02, 1, NULL },
+    { 0x0093, 0x02, 1, NULL },
+    { 0x0096, 0x02, 1, NULL },
+    { 0x00A0, 0x02, 0, NULL },
+    { 0x00C0, 0x20, 1, NULL },
+    { 0x04D0, 0x02, 0, NULL },
+    { 0x0CF8, 0x08, 1, &HaliHandlePCIConfigSpaceAccess },
+    { 0x0000, 0x00, 0, NULL }
+};
+
 /* FUNCTIONS ******************************************************************/
 
+NTSTATUS
+NTAPI
+HaliHandlePCIConfigSpaceAccess(BOOLEAN IsRead,
+                               ULONG Port,
+                               ULONG Length,
+                               PVOID Buffer)
+{
+    DPRINT1("HaliHandlePCIConfigSpaceAccess: IsRead %X, Port %X, Length %X, Buffer %p\n", IsRead, Port, Length, Buffer);
```

Comment:
Improve debug print, but I'm not sure about boolean format.
```suggestion
    DPRINT1("HaliHandlePCIConfigSpaceAccess: IsRead %X, Port 0x%X, Length %u, Buffer %p\n", IsRead, Port, Length, Buffer);
```

Reply:
`typedef unsigned char BOOLEAN, *PBOOLEAN;`

Reply:
`typedef int BOOL;`

Reply:
Committed, thanks.